crates/core/src/main.rs

```diff
@@ -23,7 +23,9 @@ static SCRIPT_NAME: &str = "script.js";
 pub extern "C" fn init() {
     unsafe {
         let mut context = Context::default();
-        context.register_globals(io::stdout()).unwrap();
+        context
+            .register_globals(io::stderr(), io::stderr())
```

Comment:
I'm sure there's a way to just pass this in once but I wouldn't figure out, happy to pair with someone on it if you know